pkg/unarchive/dmg.go

```diff
@@ -109,31 +105,27 @@ func (unarchiver *dmgUnarchiver) Unarchive(ctx context.Context, fs afero.Fs, bod
 		m = io.MultiWriter(f, bar)
 	}
 
-	err = writeDmgFile(m, body, dest)
-	if err != nil {
-		return fmt.Errorf("writeDmgFile failed :%w", err)
+	if err := writeDmgFile(m, body, dest); err != nil {
+		return fmt.Errorf("write a dmg file: %w", err)
 	}
 
 	exe := exec.New()
 	tmpMountPoint := destDir + string(filepath.Separator) + "mount"
 	_, hdiutilDetach, err := exe.HdiutilAttach(ctx, dest, tmpMountPoint)
 	if err != nil {
-		return fmt.Errorf("hdiutil attach failed: %w", err)
+		return fmt.Errorf("hdiutil attach: %w", err)
```

Comment:
In this project, basically we don't include a word `failed` in `fmt.Errorf` because it is obvious the command failed and `failed` has no meaning.